Cmake fails for Gvm-libs

Hi, I'm trying to create a Dockerfile which will install GVM10 from its source code. I'm using alpine 3.10 as base image. I'm following the install.md for each component.

### Expected behavior
cmake with success in the folder where I unzipped the source code of gvm-libs and go to the next stop which will be make and make install.

### Current behavior
Warnings stops with error and abort the process. 

### Steps to reproduce
1. Dockerfile with base image alpine:3.10
2. install dependencies
3. wget the source code in this case for the gvm-libs.
4. unzip
5. create a new folder and cd in it.
6. cmake .. 

### GVM versions
**gsa:** (gsad --version)
8.0.1
**gvm:** (gvmd --version)
8.0.1
**openvas-scanner:** (openvassd --version)
6.0.1
**gvm-libs:**
10.0.1
**openvas-smb:**
1.0.5

### Environent
**Operating system:**
Linux Alpine
**Installation method / source:** (packages, source installation)
From source code https://community.greenbone.net/t/gvm-10-old-stable-initial-release-2019-04-05/208
### Logfiles
<!-- in most cases you'll find the logs in /var/log/gvm/ -->
```
[100%] Building documentation...
[100%] Built target doc-full
[  2%] Building C object base/CMakeFiles/gvm_base_shared.dir/hosts.c.o
/root/gvm-libs-10.0.1/base/hosts.c: In function 'gvm_hosts_deduplicate':
/root/gvm-libs-10.0.1/base/hosts.c:1016:3: error: implicit declaration of function 'malloc_trim'; did you mean 'malloc'? [-Werror=implicit-function-declaration]
   malloc_trim (0);
   ^~~~~~~~~~~
   malloc
cc1: all warnings being treated as errors
make[2]: *** [base/CMakeFiles/gvm_base_shared.dir/build.make:115: base/CMakeFiles/gvm_base_shared.dir/hosts.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:123: base/CMakeFiles/gvm_base_shared.dir/all] Error 2
```

--- a/base/hosts.c
+++ b/base/hosts.c
@@ -1093,7 +1093,9 @@ gvm_hosts_deduplicate (gvm_hosts_t *hosts)
   hosts->count -= duplicates;
   hosts->duplicated += duplicates;
   hosts->current = 0;
+#ifdef __GLIBC__
   malloc_trim (0);
+#endif
 }
 
 /**
@@ -1288,7 +1290,9 @@ gvm_hosts_new_with_max (const gchar *hosts_str, unsigned int max_hosts)
     gvm_hosts_deduplicate (hosts);
 
   g_strfreev (split);
+#ifdef __GLIBC__
   malloc_trim (0);
+#endif
   return hosts;
 }
 

